src/NCronJob/Registry/IInstantJobRegistry.cs

```diff
@@ -75,6 +75,7 @@ Guid RunScheduledJob<TJob>(TimeSpan delay, object? parameter = null, Cancellatio
     /// <param name="parameter">An optional parameter that is passed down as the <see cref="JobExecutionContext"/> to the job.</param>
     /// <param name="token">An optional token to cancel the job.</param>
     /// <returns>The job correlation id.</returns>
+    [Obsolete("Use RunScheduledJob<TJob>(TimeSpan, object?, CancellationToken) instead.")]
```

Comment:
I assume that is marked "only" as a warning now?

Reply:
@linkdotnet Indeed.

And the user can either ignore it now through pragma directives or follow the recommendation and reduce the bump workload when v5 is released.